src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -296,30 +308,31 @@ WebsocketTransport.prototype.setConnectionSlowCallback = function(callback) {
     this.connectionSlowCallback = callback;
 };
 
-WebsocketTransport.prototype.getAuthorizePromise = function(contextId) {
-    if (this.isAuthorized) {
+WebsocketTransport.prototype.getAuthorizePromise = function(contextId, forceAuthenticate) {
+    if (!forceAuthenticate && this.authorizePromise) {

```

Comment:
I changed this logic - if we get a d/c around a token change we can be running 2 authorizations with the same parameters.
I can't think of how it would go wrong, but I don't like it..